src/interest/interest.service.ts

```diff
@@ -33,4 +33,16 @@ export class InterestService {
     this.logger.debug('Map interests to dto');
     return this.entityMapper.mapArray(InterestShowDto, interests);
   }
+
+  async delete(interestId: number): Promise<void> {
+    this.logger.debug('Delete an interest');
+    const interest = await this.interestRepository.findOne({
+      where: { id: interestId },
```

Comment:
Maybe we should add a catch here like in interest repository delete call

Reply:
Catch added to findOne call